judgels-backends/sandalphon/sandalphon-api/src/main/java/judgels/sandalphon/api/problem/bundle/ItemConfig.java

```diff
@@ -1,5 +1,21 @@
 package judgels.sandalphon.api.problem.bundle;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.function.Function;
+
 public interface ItemConfig {
     String getStatement();
+    ItemConfig processDisplayText(Function<String, String> processor);
+    ItemConfig withoutGradingInfo();
+
+    static ItemConfig fromString(ObjectMapper objectMapper, ItemType itemType, String string) throws IOException {
```

Comment:
Move this to the resource class (at least for now).

Reply:
Replaced with Processor pattern (like in `ScoreboardProcessor`)